monkey/monkey_island/cc/services/utils/config_encryption.py

```diff
@@ -0,0 +1,56 @@
+import base64
+import io
+import json
+import logging
+from typing import Dict, Union
+
+import pyAesCrypt
```

Comment:
This needs to be added to the `Pipfile`

Reply:
Nice catch. @shreyamalviya and I forgot that we should use pipenv instead of pip to install our requirements from now :)